dspace-api/src/test/java/org/dspace/builder/ItemBuilder.java

```diff
@@ -83,6 +83,10 @@ public ItemBuilder withAuthor(final String authorName) {
         return addMetadataValue(item, MetadataSchemaEnum.DC.getName(), "contributor", "author", authorName);
     }
 
+    public ItemBuilder withAuthor(final String authorName, final String authority) {
+        return addMetadataValue(item, DC.getName(), "contributor", "author", null, authorName, authority, 600);
```

Comment:
I'm not sure if we need this short cut as we already have a method to build an Item with an author. But after thinking about it, I have not hard opinion on this.

Reply:
@vins01-4science it looks like changes to this file can be reverted, they are unrelated with the PR purpose